### ISSUE TITLE
ampersand in url in sites.json breaks icon...

### ISSUE THREAD

**Issue by trip5** [daledavies/jump#82]
Not sure what's going on here and at first I thought something was wrong with my json... but simple enough to fix I hope?  An ampersand in the url breaks the icon... removing the & from the link allows the icon to appear.

Here are some examples where the default icon gets displayed instead of the correct icon.  BTW moving around the entries' order (ie putting the url last) doesn't change anything.

```
        {
            "name": "Calendar",
            "url" : "https://www.google.com/calendar/render?tab=wc&pli=1&hl=en",
            "icon": "calendar.svg",
            "tags": ["home"]
        },
        {
            "name": "Amazon",
            "url" : "http://www.amazon.com/b?ie=UTF8&node=230659011",
            "icon": "amazon.svg",
            "tags": ["home","shopping"]
        },
```

BTW, really like Jump.

**Comment by daledavies** [comment]
Howdy, I'll take a look to see if I can recreate this.

Thanks for reaching out and for the kind words about Jump 😃

**Comment by daledavies** [comment]
Thanks for reporting this, I found the bug and fixed it. The patch will be bundled into the next release :)

**Comment by darkpoet78** [comment]
Awesome.  Thanks so much.  Just waiting on your docker push.  ^^

**Comment by daledavies** [comment]
Just have to write the docs for a few new features in the next release, hoping to do it this weekend 😃

**Comment by daledavies** [comment]
I know it has been a while but this is now available in v1.4.x so I'm closing this issue. Of course feel free to reopen if you need to. Thanks for helping out 😁
